components/qrcode/index.tsx

```diff
@@ -35,7 +36,7 @@ const QRCode: React.FC<QRCodeProps> = (props) => {
   const prefixCls = getPrefixCls('qrcode', customizePrefixCls);
   const [wrapSSR, hashId] = useStyle(prefixCls);
   const { token } = useToken();
-  const qrCodeProps = useMemo<QRPropsCanvas>(() => {
+  const qrCodeProps = useMemo<QRPropsCanvas | QRPropsSvg>(() => {
```

Comment:
```suggestion
  const qrCodeProps = useMemo<QRProps>(() => {
```
这里用 `QRProps` 吧，下面就不需要断言了，记得 export 一下